Update documentation to match new alias command changes (#81)
eik-lib/eik-lib.github.io: docs/client_app_packages.md

```json
{
  "files": "./dist"
}
```

#### package.json file entrypoints

```json
{
  "eik": {
    "files": "./dist"
  }
}
```

### The publish command

> With entrypoints defined in the Eik config, running the `eik publish` command will assemble files (specified by entrypoints) into an archive and upload the archive to the Eik server defined by the `server` field. The type of package uploaded depends on the `type` field specified in the config, which defalts to `package`, but other possible values are `npm` and `map`.

Reviewer: ```suggestion
With entrypoints defined in the Eik config, running the `eik publish` command will assemble files (specified by entrypoints) into an archive and upload the archive to the Eik server defined by the `server` field. The type of package uploaded depends on the `type` field specified in the config. Possible values are `package`, `npm` and `map` with `package` being the default.
```
